src/ray/gcs/gcs_server/gcs_table_storage.cc

```diff
@@ -52,7 +52,7 @@ Status GcsTable<Key, Data>::GetAll(const MapCallback<Key, Data> &callback) {
       if (!item.second.empty()) {
         Data data;
         data.ParseFromString(item.second);
-        values[Key::FromBinary(item.first)] = data;
+        values[Key::FromBinary(item.first)] = std::move(data);
```

Comment:
How about just one line? This will save the memory copy from stack to heap.

```cpp
values[Key::FromBinary(item.first)].ParseFromString(item.second);
```

Reply:
Sounds great! It's been rephrase and so are those in `GetByJobID`.